fotoapparat/src/test/java/io/fotoapparat/parameter/selector/PreviewFpsFrameSelectorsTest.java

```diff
@@ -0,0 +1,142 @@
+package io.fotoapparat.parameter.selector;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.Arrays;
+import java.util.List;
+
+import io.fotoapparat.parameter.range.IntervalRange;
+import io.fotoapparat.parameter.range.Range;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+@RunWith(MockitoJUnitRunner.class)
+public class PreviewFpsFrameSelectorsTest {
```

Comment:
Should be `PreviewFpsRangeSelectorsTest`, so that it includes the name of the class which we are testing.